.github/Dockerfile-llvm

```
@@ -0,0 +1,16 @@
+FROM alpine:3.20.3
+
+ARG RUST_VERSION=stable
+
+RUN apk add bash git cmake make g++ ninja python3 ncurses-static curl lld-dev lld lld-libs libc++-static
+RUN ninja --version
+
+COPY build-llvm.sh .
+
+RUN bash -c ./build-llvm.sh

```

```suggestion
RUN bash -c ./build-llvm.sh
```

We probably need to modify the `build-llvm.sh` file to use other flags. AFAIK @xermicus discovered that some other flags are needed to it statically links against LLVMs `libc++` instead of gccs  `libstdc++`.

---

Yes, when I tried cross compiling to musl using the current build script, resolc was still linked against `libstdc++` (and segfaulted with an LLVM panic message). See also this [comment in the related issue](https://github.com/paritytech/revive/issues/93#issuecomment-2423776917). 

@wpt967 mentioned he was looking into it, what is the status?

---

I told @wpt967 to de-pritotize this one since we are only supporting compilation REMIX for now. Until we have proper releases.